sys/include/crypto/padding.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2017 Mathias Tausig
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_crypto
+ * @{
+ *
+ * @file        padding.h
+ * @brief       Padding modes for encryption
+ *
+ * @author      Mathias Tausig
+ */
+
+#ifndef CRYPTO_PADDING_H
+#define CRYPTO_PADDING_H
+
+#include "crypto/ciphers.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
```

Comment:
please move this above ```#include "crypto/ciphers.h"```

Reply:
Done 2233d6c4ba5db0371133842ffd8508e47569c875